pkgs/development/python-modules/pyworld/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib
+, buildPythonPackage, fetchPypi
+, numpy, cython
+}:
+
+buildPythonPackage rec {
+  pname = "pyworld";
+  version = "0.2.8";
+
+  src = fetchPypi {
```

Comment:
The PyPI tarball does't contain any tests, hence `0 tests` will be logged in the build output. If there are any tests in the Git tarball (e.g. from GitHub), it's fine to use that one to have some tests running during the build. If there aren't any tests (or mostly impure ones that will break during `nix-build`), please set `doCheck = false`.